docs/core/extensions/httpclient-http3.md

```diff
@@ -54,36 +38,48 @@ HTTP/3 uses QUIC as its transport protocol. The .NET implementation of HTTP/3 us
 
 ### Linux
 
+- OpenSSL 1.1
+
 On Linux, libmsquic is published via Microsoft's official Linux package repository packages.microsoft.com. To consume it, it must be added manually. See [Linux Software Repository for Microsoft Products](/windows-server/administration/linux-package-repository-for-microsoft-software). After configuring the package feed, it can be installed via the package manager of your distro, for example, for Ubuntu:
 
 ```bash
-sudo apt install libmsquic=1.9*
+sudo apt install libmsquic
 ```
 
 > [!NOTE]
-> .NET 6 is only compatible with the 1.9.x versions of libmsquic. Libmsquic 2.x is not compatible due to breaking changes. Libmsquic will receive updates to 1.9.x when needed to incorporate security fixes.  
+> .NET 7 is only compatible with 2.1+ versions of libmsquic.
 
 ### macOS
 
 HTTP/3 is not currently supported on macOS but may be available in a future release.
 
 ## Using HttpClient
 
-Include the following in the project file to enable HTTP/3 with HttpClient:
+The following code example uses [top-level statements](../../csharp/fundamentals/program-structure/top-level-statements.md) and demonstrates how to specify HTTP3 in the request:
+
+:::code language="csharp" source="snippets/http/http3/Program.cs":::
+
+## HTTP/3 Support in .NET 6
+
+In .NET 6, HTTP/3 is available as a _preview feature_ because the HTTP/3 specification was not yet finalized and behavioral or performance problems may exist in HTTP/3 with .NET 6. For more information on preview features, see [**the preview features specification**](https://github.com/dotnet/designs/blob/main/accepted/2021/preview-features/preview-features.md#are-preview-features-supported).
```

Comment:
```suggestion
In .NET 6, HTTP/3 is available as a _preview feature_ because the HTTP/3 specification was not yet finalized. Behavioral or performance problems may exist in HTTP/3 with .NET 6. For more information on preview features, see [the preview features specification](https://github.com/dotnet/designs/blob/main/accepted/2021/preview-features/preview-features.md#are-preview-features-supported).
```